openshift_tools/cloud/aws/ebs_snapshotter.py

```diff
@@ -301,6 +306,7 @@ def get_expired_snapshots(snapshots, hourly_backups, daily_backups, weekly_backu
                     #date the date ranges and the snapshots
                     # are sorted chronologically, we know this
                     #snapshot isn't in an earlier date range):
+                    # pylint: disable=singleton-comparison
```

Comment:
Please add a reason block here per our best practices:
https://github.com/openshift/openshift-tools/blob/prod/docs/best_practices_guide.adoc#All-PyLint-rule-disables-MUST-be-documented-in-the-code